src/main/java/uk/gov/ons/census/fwmt/jobservice/service/routing/hh/HhUpdateEnglandAndWales.java

```diff
@@ -0,0 +1,98 @@
+import uk.gov.ons.census.fwmt.jobservice.service.routing.RoutingValidator;
+
+import static uk.gov.ons.census.fwmt.jobservice.config.GatewayEventsConfig.COMET_DELETE_ACK;
+import static uk.gov.ons.census.fwmt.jobservice.config.GatewayEventsConfig.COMET_DELETE_PRE_SENDING;
+import static uk.gov.ons.census.fwmt.jobservice.config.GatewayEventsConfig.COMET_UPDATE_ACK;
+import static uk.gov.ons.census.fwmt.jobservice.config.GatewayEventsConfig.COMET_UPDATE_PRE_SENDING;
+import static uk.gov.ons.census.fwmt.jobservice.config.GatewayEventsConfig.FAILED_TO_CREATE_TM_JOB;
+
+@Qualifier("Update")
+@Service
+public class HhUpdateEnglandAndWales implements InboundProcessor<FwmtActionInstruction> {
+
+  @Autowired
+  private CometRestClient cometRestClient;
+
+  @Autowired
+  private GatewayEventManager eventManager;
+
+  @Autowired
+  private RoutingValidator routingValidator;
+
+  @Autowired
+  private GatewayCacheService cacheService;
+
+  private static final ProcessorKey key = ProcessorKey.builder()
+      .actionInstruction(ActionInstructionType.CREATE.toString())
+      .surveyName("CENSUS")
+      .addressType("HH")
+      .build();
+
+  @Override
+  public ProcessorKey getKey() {
+    return key;
+  }
+
+  @Override
+  public boolean isValid(FwmtActionInstruction rmRequest, GatewayCache cache) {
+    try {
+      return rmRequest.getActionInstruction() == ActionInstructionType.UPDATE
+          && rmRequest.getSurveyName().equals("CENSUS")
+          && rmRequest.getAddressType().equals("HH")
+          && !rmRequest.getOa().startsWith("N")
+          && (cache != null && cache.existsInFwmt);
+    } catch (NullPointerException e) {
+      return false;
+    }
+  }
+
+  @Override
+  public void process(FwmtActionInstruction rmRequest, GatewayCache cache) throws GatewayException {
+    CaseRequest tmRequest = HhCreateConverter.convertHhEnglandAndWales(rmRequest, cache);
+
+    eventManager.triggerEvent(String.valueOf(rmRequest.getCaseId()), COMET_UPDATE_PRE_SENDING,
+        "Case Ref", tmRequest.getReference(),
+        "Survey Type", tmRequest.getSurveyType().toString());
+
+    ResponseEntity<Void> response = cometRestClient.sendCreate(tmRequest, rmRequest.getCaseId());
+    routingValidator.validateResponseCode(response, rmRequest.getCaseId(), "Update", FAILED_TO_CREATE_TM_JOB);
+
+    eventManager
+        .triggerEvent(String.valueOf(rmRequest.getCaseId()), COMET_UPDATE_ACK,
+            "Case Ref", rmRequest.getCaseRef(),
+            "Response Code", response.getStatusCode().name(),
+            "Survey Type", tmRequest.getSurveyType().toString());
+
+    if (!rmRequest.isBlankFormReturned() || !rmRequest.isUndeliveredAsAddress()) {
+      eventManager.triggerEvent(String.valueOf(rmRequest.getCaseId()), COMET_DELETE_PRE_SENDING,
+          "Case Ref", tmRequest.getReference(),
+          "Survey Type", tmRequest.getSurveyType().toString());
+
+      response = cometRestClient.sendDelete(rmRequest.getCaseId());
```

Comment:
This needs to be a new call called `deletePause`. Its not a delete of the case, change the trigger event name aswell